dataeng/resources/prefect-flows-deployment.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -ex
+
+# Creating python3.8 virtual env
+PLATFORM_VENV="platform_venv"
+virtualenv --python=python3.8 --clear "${PLATFORM_VENV}"
+source "${PLATFORM_VENV}/bin/activate"
+
+# Removing prefix 'prefect-flows-deployment-'
+# $FLOW will contain the name of flow
+FLOW=$(echo $JOB_NAME | cut -c 26-)
+
+cd $WORKSPACE/edx-prefectutils
+pip install -r requirements/base.txt
+pip install -r requirements/base.in
+
```

Comment:
`install -r requirements/base.txt`
`pip install -r requirements/base.in`

I am installing these two as a requirements here but there might be a better way do it

Reply:
I thought `base.in` should be installed, I am installing `base.in` in setup.py file as well, should it be changed to `base.txt` 
[setup.py#L92](https://github.com/edx/edx-prefectutils/blob/master/setup.py#L92)

Reply:
Everything in `base.in` is contained in `base.txt`. The difference is that `base.txt` has the pinned versions for these requirements, as well as all dependencies. Running `pip install` for both is redundant.